qaseio/changelog.md

```diff
@@ -1,3 +1,9 @@
+# qaseio@2.1.0-beta.1
+
+## What's new
+
+* Added support for uploading attachments from strings and buffers. 
```

Comment:
It's the end product, so we should tell about things that changed for the users. 

```suggestion
Upload test attachments to Qase.

```js
test('test', async ({ page }) => {
  // upload files by path
  qase.attach({ paths: '/path/to/file'});
  // list multiple files at once
  qase.attach({ paths: ['/path/to/file', '/path/to/another/file']});
  // upload contents directly from your code
  qase.attach({ name: 'attachment.txt', content: 'Hello, world!', contentType: 'text/plain' });
  await page.goto('https://example.com');
 });
```

When passing custom data from the code, specify `contentType` explicitly.
Most popular file extensions have their content type determined automatically:   `.css`, `.eot`, `.gif`, `.html`, `.jpg`, `.js`, `.json`, `.mp4`, `.otf`, `.png`, `.svg`, `.ttf`, `.wasm`, `.wav`, and `.woff`.

Reply:
I will add mime-types package and move this logic to qase-javascript-commons next PR.